CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ set(target_specific_public_libraries "")
 set(target_specific_private_definitions "")
 set(target_specific_dependencies "")
 
+# Build options
+option(USE_HARFBUZZ "Use Harfbuzz" ON) #ON by default
```

Comment:
I don't think we need this option

Reply:
It is practice that we could follow  - use `options` - make it easy to configure.

Reply:
Have you seen it being used anywhere in project? Please keep PR changes related to the PR subject and miniman

Reply:
Also, `#ON by default` is useless comment, `int a = 5; // assign value 5`-style

Reply:
Since you brought up YAGNI in other thread, won't same principle apply to this?

Reply:
`cmake` configuration for the text rendering by the `Harfbuzz` have been omitted.